doc/releasenotes/15_0_0_summary.md

```diff
@@ -17,6 +17,8 @@
 
 ### Breaking Changes
 
+- The deprecated `--cpu_profile` flag has been removed. Please use the `--pprof` flag instead.
```

Comment:
Two other flags were deleted. Can you please document those and then merge?
```
       --mem-profile-rate int                                             deprecated: use '-pprof=mem' instead (default 524288)
       --mutex-profile-fraction int                                       deprecated: use '-pprof=mutex' instead
```

Reply:
oh whoops, yep!!